src/plugins/intel_gpu/src/graph/impls/cpu/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "openvino_intel_gpu_cpu_obj")
+
+ov_gpu_add_backend_target(
+    NAME ${TARGET_NAME}
+    ROOT ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
if you replace cmake's `function` to `macro`, you don't have to define `ROOT` and can use `${CMAKE_CURRENT_SOURCE_DIR}` directly within a macro